src/assets/css/style.css

```diff
@@ -1339,6 +1339,7 @@ table.manage-table td.action a:hover {
   right: 0;
   bottom: 20px;
   margin: 0 20px 0 0;
+  z-index: 1000;
```

Comment:
Hi,
Set `z-index` to 1000 without counting could be a bad approach. Did you calculate it or is the value for random?. In my opinion, conforming to the value with the best value will be the descriptor.

Reply:
This is a back to top button so it can have the highest z-index.